### PR TITLE
Complete nginx config,

### PR DESCRIPTION
**Pull request by cxmcc**

### CODE REVIEW

File: salt/nginx.conf [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
+      client_max_body_size 1M;
 
-  location / {
-    proxy_set_header X-Real-IP $remote_addr;
-    proxy_set_header HOST $http_host;
-    proxy_set_header X-NginX-Proxy true;
+      location / {
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header HOST $http_host;
+        proxy_set_header X-NginX-Proxy true;
```
Comment (nickelser) [inline salt/nginx.conf]: what's up with these weird header names and capitalization?

File: salt/nginx.conf [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
+      server 127.0.0.1:8000;
+      server 127.0.0.1:8001;
+      server 127.0.0.1:8002;
+      server 127.0.0.1:8003;
+    }
+
+    server {
+      listen       80;
+      server_name  _;
+      client_max_body_size 1M;
```
Comment (nickelser) [inline salt/nginx.conf]: is this big enough for all images?

File: salt/nginx.conf (hunk 1 above)
Comment (cxmcc) [inline salt/nginx.conf]: @ptrln

Comment (solidsnack) [inline salt/nginx.conf]: Yeah, `HOST` should be `Host`. I am curious as to why we are setting `X-NginX-Proxy`.

Comment (ptrln) [inline salt/nginx.conf]: I literally just copy and paste this file. @cxmcc said we actually don't need any of these proxy_set_headers

File: salt/nginx.conf (hunk 2 above)
Comment (ptrln) [inline salt/nginx.conf]: I think the large images are 600x600, which are very very very unlikely to be > 1M.

**Pull request merged**